tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
My argument is that by writing _any_ way of copying a class we've already defined what it means for two instances of that class to be equal. Be it copy construction (assignment) or a `clone` or `get_clone` function, the meaning of equality is already defined. If we think that the definition of copy does not satisfy our definition of equality then we should rethink the copy semantics (here I mean copy in the sense of constructors, assignment operators, or cloning functions). Now, one may argue that we should never need to compare instances of a specific class, in which case my response would be "Why should we be able to copy it then?" For abstract base classes the implementation becomes more difficult and there I can see the argument for saying, "We should never need to compare these objects and the implementation of comparison is not trivial so let's omit it."